### ISSUE TITLE
Bootstrap5 alpha3: scale-color function conflicts with built in sass scale-color

### ISSUE THREAD

**Issue by russellrazor** [twbs/bootstrap#32140]
**Operating system and version (Windows, macOS, Android, iOS)**
macOS 10.15.7 and docker `node:lts`

**Browser and version (Chrome, Firefox, Safari, Microsoft Edge, Opera, Android Browser)**
N/A

**Bootstrap Version**
Bootstrap 5 alpha3

**Issue**
`scale-color` function conflicts with built in sass `scale-color`
https://sass-lang.com/documentation/modules/color#scale

The scale-color signature for the built in method is:
```scss
scale-color($color,
  $red: null, $green: null, $blue: null,
  $saturation: null, $lightness: null,
  $whiteness: null, $blackness: null,
  $alpha: null
)
```

Bootstrap5 alpha3 signature is:
```scss
scale-color($color, $weight)
```

These are incompatible and break scss builds.

**Error logs:**
```
#15 100.6 SassError: Missing argument $weight.03:03
#15 100.6     ┌──> src/styles/mixins/_colours.scss03:03
#15 100.6 2   │     $col-light: scale-color($colour, $lightness: +15%);03:03
#15 100.6     │                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ invocation03:03
#15 100.6     ╵03:03
#15 100.6     ┌──> node_modules/bootstrap/scss/_functions.scss03:03
#15 100.6 166 │ @function scale-color($color, $weight) {03:03
#15 100.6     │           ━━━━━━━━━━━━━━━━━━━━━━━━━━━━ declaration03:03
#15 100.6     ╵03:03
#15 100.6   src/styles/mixins/_colours.scss 2:14  scale-color()
```

Relates to: #31538

**Comment by XhmikosR** [comment]
Yeah, it's already known see #32107. Not sure if we will prefix all functions, but we need to fix it. Renaming the offending function should be enough and should reduce the breaking changes going from v4 to v5.

/CC @twbs/css-review

**Comment by XhmikosR** [comment]
Fixed by #32149
